package.json

```diff
@@ -11,16 +11,17 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@radix-ui/react-dropdown-menu": "^2.0.6",
```

Comment:
I don't think this dependency is used anywhere in the code

Reply:
I intented to use it but i found another options. I'll remove it as am also working on the documentation.

Reply:
Sounds good. Thanks! Please also update the .mdx files and add the details for this component.

Reply:
Sure.👍